### PR TITLE
New: replaceOccurrencesOfString:(CPString)target withString:(CPString…

### PR DESCRIPTION
**Pull request by daboe01**
…)replacement options:(int)options range:(CSRange)searchRange

this implementation was missing in CPMutableAttributedString

### CODE REVIEW

**Comment by daboe01** [comment]
-#new
+#Foundation

**Comment by daboe01** [comment]
-#new
+Foundation

File: Foundation/CPAttributedString.j [hunk 1]
```diff
@@ -887,6 +887,30 @@ var CPAttributedStringStringKey     = "CPAttributedStringString",
 */
 @implementation CPMutableAttributedString : CPAttributedString
 
+/*!
+    Replaces all occurrences of the target string in a given range with a replacement string. Returns the number of replacements.
+ */
+
+- (int)replaceOccurrencesOfString:(CPString)target
+                       withString:(CPString)replacement
+                          options:(int)options
```
[the remark this answers is not in the document]
Comment (daboe01) [reply Foundation/CPAttributedString.j]: thank you, fixed

File: Foundation/CPAttributedString.j [hunk 2]
```diff
@@ -887,6 +887,30 @@ var CPAttributedStringStringKey     = "CPAttributedStringString",
 */
 @implementation CPMutableAttributedString : CPAttributedString
 
+/*!
+    Replaces all occurrences of the target string in a given range with a replacement string. Returns the number of replacements.
+ */
+
+- (int)replaceOccurrencesOfString:(CPString)target
```
[the remark this answers is not in the document]
Comment (daboe01) [reply Foundation/CPAttributedString.j]: thank you, fixed

**Comment by mrcarlberg** [comment]
Merged! Thanks!

**Pull request merged**